lib/src/date_box.dart

```diff
@@ -30,8 +34,21 @@ class DateBox extends StatelessWidget {
   /// Pressed callback function.
   final VoidCallback? onPressed;
 
+  /// Show DateBox event in container.
+  final bool showDot;
+
+  /// DateBox is today.
+  final bool isToday;
+
+  /// DateBox selection.
+  final bool isSelected;
+
+  /// Show event in DateBox.
+  final bool hasEvent;
```

Comment:
Those changes should be declared in the README.md documentation file, coz it's a community lib.